src/star.css

```diff
@@ -0,0 +1,21 @@
+
+.stars {
```

Comment:
you don't need this blank line at the beginning of your files, fix it everywhere